docs/developer-docs/defi/rosetta/icrc_rosetta/construction_api/index.md

```diff
@@ -0,0 +1,354 @@
+---
+keywords: [intermediate, rosetta, tutorial]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Construction-API implementation
+
+<MarkdownChipRow labels={["Intermediate", "Tutorial" ]} />
+
+## Overview 
+
+The Construction-API allows for offline signing of transactions and then posting them on ICP at a later point in time. It is recommended to first go through [flow of operations](/docs/developer-docs/defi/rosetta/icp_rosetta/construction_api/operations_flow/index.mdx) of the Construction-API to gain an understanding of how it works, what endpoints to call with what parameters, and where the user has to take action upon receiving certain responses. 
+
+Currently, the Construction-API of Rosetta supports only one category of transactions, which is the transfer of assets. According to the ICRC-1 and ICRC-2 standards this can be done in the following ways:
+- [Direct transfers](/docs/developer-docs/defi/rosetta/icrc_rosetta/construction_api/asset_transfer/transfer.mdx): This is the most basic functionality. In accordance with the ICRC-1 standard you can use Rosetta to transfer ICP. 
+- [Approve spender](/docs/developer-docs/defi/rosetta/icrc_rosetta/construction_api/asset_transfer/approve.mdx): You can approve a spender to spend your tokens for you. This feature adheres to the ICRC-2 standard. 
+
+## Example implementation 
+For the sake of completion, the entire Rosetta-Client sample implementation in this example lists all functionalities of the Construction-API. 
```

Comment:
```suggestion
# Construction API implementation

<MarkdownChipRow labels={["Intermediate", "Tutorial" ]} />

## Overview 

The construction API allows for offline signing of transactions and then posting them on ICP at a later point in time. It is recommended to first go through [flow of operations](/docs/developer-docs/defi/rosetta/icp_rosetta/construction_api/operations_flow/index.mdx) of the construction API to gain an understanding of how it works, what endpoints to call with what parameters, and where the user has to take action upon receiving certain responses. 

Currently, the construction API of Rosetta supports only one category of transactions, which is the transfer of assets. According to the ICRC-1 and ICRC-2 standards this can be done in the following ways:
- [Direct transfers](/docs/developer-docs/defi/rosetta/icrc_rosetta/construction_api/asset_transfer/transfer.mdx): This is the most basic functionality. In accordance with the ICRC-1 standard you can use Rosetta to transfer ICP. 
- [Approve spender](/docs/developer-docs/defi/rosetta/icrc_rosetta/construction_api/asset_transfer/approve.mdx): You can approve a spender to spend your tokens for you. This feature adheres to the ICRC-2 standard. 

## Example implementation 
For the sake of completion, the entire Rosetta-Client sample implementation in this example lists all functionalities of the construction API. 
```